mypy/checker.py

```diff
@@ -796,6 +796,7 @@ def check_reverse_op_method(self, defn: FuncItem, typ: CallableType,
                 arg_types = list(arg_type.iter_deep())
             else:
                 return
+            # We check that each method is fine when dispatched on a proper self argument
```

Comment:
FYI you can make your local commit history more relevant by using `git commit --amend` for such things (since you did not push to GitHub in between).